### DISCUSSION TITLE
How-To: Use zellij to start processes during boot

### DISCUSSION THREAD

**Discussion by smezger** [zellij-org/zellij#2698] [Q&A]
I'd like to use cron/zellij to start some processes during boot. 
Here's what I've tried:
crontab: `@reboot zellij -l layout.kdl`
`layout.kdl `contains the processes to run

However after boot no zellij sessions are running and log does not indicate any error.
Am I missing something?  Thanks for your help!

**Comment by Lockszmith-GH** [comment]
I seems to me you're thinking of running zellij and placing it in the background.
Are you hoping for the session to be ready for you when you login?

If you want to join a running zellij session, you first must have that zellij session run as the same user as the active terminal connection.

I'm unsure if tty is required, but you might need to research how to open a tty for zellij.

**Reply by smezger** [reply]
@Lockszmith-GH : Thanks for your reply! 
Yes I try to achieve to have the session ready when I login, as the cronjob is configured with the same user as I use later, the user does not seem to be the problem. 
I had a similar setup running with tmux, also if I encapsulate the zellij start from cronjob in a tmux session I achieve running zellij in tmux. 
```
tmux new -d -s basement-pi \; rename-window zellij \;
tmux send-keys -t basement-pi:zellij "zellij -l /home/pi/layout.kdl -s basement-pi" ENTER
```
Doing something similar for zellij i.e using CLI to control is not possible as zellij has no option to start a new session in detached mode (or I'm struggling to find how). After that it might be possible to use `zellij -s session-name action new-tab -l layout.kdl`

Encapsulating the start of zellij in a bash script called from the cronjob @boot doesn't help either.

**Reply by Lockszmith-GH** [reply]
I think you are right, that option to start, then detach, doesn't exist yet.

#1263 from a year ago seems to have that requested.
